pkg/util/configuration/configuration.go

```diff
@@ -49,12 +51,30 @@ func GetSidecarSetPatchMetadataWhiteList(client client.Client) (*SidecarSetPatch
 	return whiteList, nil
 }
 
+func GetPPSWatchWatchCustomWorkloadWhiteList(client client.Client) (*PPSWatchWatchCustomWorkloadWhiteList, error) {
+	whiteList := &PPSWatchWatchCustomWorkloadWhiteList{Workloads: make([]schema.GroupVersionKind, 0)}
+	data, err := getKruiseConfiguration(client)
```

Comment:
Maybe, we can just initialize the whiteList in a `init` function at the moment of  kruise starting?